src/tape_drivers/freebsd/cam/cam_tc.c

```diff
@@ -1244,7 +1244,7 @@ static int camtape_get_next_block_to_xfer(void *device, struct tc_position *pos)
 		camtape_process_errors(softc, rc, msg, "READPOS", true);
 	else {
 		pos->partition = ext_data.partition;
-		pos->block = scsi_8btou64(ext_data.last_object)
+		pos->block = scsi_8btou64(ext_data.last_object); // here on line no. 1247 there was semicolon missing --> so i fixed it
```

Comment:
Oops, could you remove the comment on line 1247?

The comment must be added to the logical explanation of the code itself.

Reply:
sorry i was busy so didn't see the msg